onecodex/lib/upload.py

```diff
@@ -216,7 +197,11 @@ def cancel_atexit():
             log.info(f"Canceled upload for {filename} as sample {fields['sample_id']}")
 
             try:
-                samples_resource.cancel_upload({"sample_id": fields["sample_id"]})
+                resp = samples_resource._client.post(
```

Comment:
`_client` and `_api` are named as a protected members of resource class. Are they? Or are special but public attributes? If so then I am not sure what are best practices in Python to have special attributes.

----

If they are protected members then I probably would bringing back `cancel_upload` method (and `init_multipart_upload` from above).